### PR TITLE
Add warning to HELP.md when user requested maintenance mode dependency

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**
Fixes gh-28

### CODE REVIEW

**Comment by mbhave** [comment]
Thanks for the PR @OlgaMaciaszek. I think we should also add the recommended replacement for the maintenance mode dependency instead of just saying `We do not recommend adding them to
new projects`

File: start-site/src/main/resources/templates/maintenance-mode-dependencies.mustache [hunk 1]
```diff
@@ -0,0 +1,6 @@
+The dependencies listed below are in maintenance mode. We do not recommend adding them to
```
Comment (snicoll) [inline start-site/src/main/resources/templates/maintenance-mode-dependencies.mustache]: I'd start with something tailored for the spring cloud use cases with a general section that titles exactly what happens and then some blurp that summarizes the situation. Perhaps a link to blog post announcement?

**Comment by OlgaMaciaszek** [comment]
Hi, @snicoll @mbhave thanks for the review. Following your suggestions, I have changed it into something specific to the Spring Cloud Netflix maintenance mode situation. I have also added link to the blog entry that has more details on the background of the maintenance mode decision, as well as the list of suggested replacement. Let me know if it's fine now.

**Comment by snicoll** [comment]
Thanks @OlgaMaciaszek - this is now merged with a [polish commit](https://github.com/spring-io/start.spring.io/commit/1d0c874933ff9dc82c31d50ab93fcbfaa8e0b680).

**Pull request closed without merging**